### DISCUSSION TITLE
2024.12.04

### DISCUSSION THREAD

**Discussion by vidalvanbergen** [vidalvanbergen/FightClub5eXML#9] [Announcements]
**Suggested downloads:**  
PHB 2014: [Complete_Compendium.zip](https://github.com/vidalvanbergen/FightClub5eXML/releases/download/2024.12.04/Complete_Compendium.zip)
PHB 2024: [Complete_Compendium_2024.zip](https://github.com/vidalvanbergen/FightClub5eXML/releases/download/2024.12.04/Complete_Compendium_2024.zip) (Only PHB for now)
PHB 2014+2024: [Complete_Compendium_2014+2024.zip]()

## What's Changed:
* Added `Dungeon Master's Guide 2024` content by @TheArchivist.
* Updated `Player's Handbook 2024` content, which are now tagged with [2024].
* Added a mixed complete compendium 2014+2024 collection with the new content tagged with [2024].
* Added Homebrew content by @reccho:
  * Added `Winghorn Press: The Wild Sheep Chase`.
  * Added `Winghorn Press: The Wolves of Welton`.
  * Added `Winghorn Press: The Low Crater`.
  * Added `Winghorn Press: A Most Potent Brew`.
  * Added `Tony Petrecca: Journey to the Center of the Underdark`.
  * Added `Tony Petrecca: Journey to the Center of the Underdark 2: The Darklake Strikes Back`.
  * Added `Tony Petrecca: Killer Kobolds!`.
  * Added `Benjamin Huffman: Pugilist Class`.
  * Added `Monsters without Borders v1.2`.
  * Added `Cawood: Monsters of Feyland`.
  * Added `Cawood: Monsters of the Underworld`.
  * Added `Cawood: Monsters of the City: Sins & Virtues`.
  * Added `Cawood: Monsters of the Wilderness: Oswald's Curse`.
  * Added `Cawood: Monsters of Feyland 2: The Well of Dreams`.
  * Added `Cawood: Monsters of the Dungeon: Brave the Darkness`.
  * Added various homebrew by `DMDave`.
  * Added `Shmeckerel - Gourdon Applegrove and Jack`.
  * Added `Spectre Creations`.
  * Added `Stormchasers`.
  * Added `Team Superhydra: The Kobolds' Cauldron`.
* Tagged `Vessel v2.2.0`, and Added `Vessel v3.0.1` by LaserLlama.
* Other changes and fixes.

**Full Changelog**: https://github.com/vidalvanbergen/FightClub5eXML/compare/2024.11.22...2024.12.04

<hr /><em>This discussion was created from the release <a href='https://github.com/vidalvanbergen/FightClub5eXML/releases/tag/2024.12.04'>2024.12.04</a>.</em>

**Comment by Catkingman** [comment]
May I request that the next source to be added be Valda's Spire of Secrets?
